PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -493,8 +493,9 @@
 run2_miniAOD_80XLegacy.toModify( fatJetTable.variables, n2b1 = None)
 run2_miniAOD_80XLegacy.toModify( fatJetTable.variables, n3b1 = None)
 
-for modifier in run2_nanoAOD_106Xv1, run2_nanoAOD_devel:
-  modifier.toModify( fatJetTable.variables, nConsistuent = None)
+
+(run2_nanoAOD_106Xv1 & ~run2_nanoAOD_devel).toModify( fatJetTable.variables, nConsistuent = None)
```

Comment:
typo !
nConsistuent --> nConstituents

good practice before push, please run locally

Reply:
@laurenhay

Reply:
Sorry for the extreme delay. I can only promise to be better about backports in the future.